firebase-crashlytics/src/androidTest/java/com/google/firebase/crashlytics/internal/ProviderProxyNativeComponentTest.java

```diff
@@ -19,6 +19,7 @@
 import androidx.test.runner.AndroidJUnit4;
 import org.junit.Before;
 import org.junit.Test;
+import org.junit.runner.RunWith;
```

Comment:
Unrelated fix for compiler error when building the android tests. Full fix in https://github.com/firebase/firebase-android-sdk/pull/2609